libs/ledger-live-common/src/families/algorand/account.ts

```diff
@@ -24,8 +25,8 @@ function formatOperationSpecifics(
     : "";
 }
 
-function formatAccountSpecifics(account: AlgorandAccount): string {
-  const { algorandResources } = account;
+function formatAccountSpecifics(account: Account): string {
```

Comment:
why this change here ?
I feel like it makes sense to have an AlgorandAccount in the signature
And we can still call the function with

```ts
const account = .... as AlgorandAccount;
formatAccountSpecifics(account);
```

it makes things easier to debug after because we can tell we passed an AlgorandAccount to the function

Reply:
@lvndry I've no strong opinion about "Account" or "AlgorandAccount".
I've just checked the codebase. It seems that it doesn't work with "AlgorandAccount", becase the "formatAccountSpecifics" method is called the common part from LLC.
At this level, we can't distinguish whether it is "AlgorandAccount" or Account of other currencies. So we need to provide a unique interface for formatAccountSpecifics.
Otherwise, it doesn't compile.

Reply:
Alright, the big weakness of this is that in theory we could call the function with:

```ts
const myBitcoinAccount = ....; // bitcoin account 
formatAccountSpecifics(account); // imported from algorand
```

And typescript will not show any error bitcoin bitcoin account extends type account.

Cant we change the signature with something like this in LLC ?

```ts
formatAccountSpecifics<T extends Account>(account: T)
```

Reply:
Will be treated outside this PR